testing/camunda-process-test-java/src/test/java/io/camunda/process/test/impl/extensions/CamundaProcessTestContextIT.java

```diff
@@ -830,7 +831,50 @@ void shouldMockBusinessRule() {
     // Then
     assertThat(processInstanceEvent).isCompleted();
     final Map<String, Object> expectedVariables = new HashMap<>();
-    expectedVariables.put("result", resultValues);
+    expectedVariables.put("result", decisionOutput);
+    assertThat(processInstanceEvent).hasVariables(expectedVariables);
+  }
+
+  @Test
+  void shouldMockBusinessRuleWithGenericOutput() {
```

Comment:
:wrench: In the test cases, we call `mockDmnDecision()` only with a map object. To increase the coverage, we could pass different object types.